sources/packages/backend/apps/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -123,7 +123,10 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
    */
   async getPendingAndDeniedAppeals(
     applicationId: number,
-    studentId?: number,
+    options?: {
+      studentId?: number;
+      limit?: number;
```

Comment:
please update the comment

Reply:
Changes will be reverted as they are no longer needed for this ticket.

Reply:
Changed.